index.js

```diff
@@ -730,27 +733,36 @@ var tick = function() {
 				if(!("units" in node)) {
 					node.units = {};
 				}
-				if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
-					if(node.owner != -1) {
-						nodeWinner = node.owner;
+				if (node.owner == -3) {
+					for (var owner in node.units) {
+						// convert all units to energy
+						gd.data.energy[owner] = Math.max(gd.data.energy[owner] + node.units[owner], GAMERULES.MAX_ENERGY);
+						node.units[owner] = 0;
+						logins[gd.users[adjustForRemoved(gd, i)]].conn.send("energy:" + gd.data.energy[owner]);
 					}
-				}
-				else {
-					if(gd.debug) {
-						console.log("contested due to node owner");
-						console.log(nodeWinner, node.owner, typeof nodeWinner, typeof node.owner);
+				} else {
+					if(nodeWinner == node.owner || node.owner == -1 || nodeWinner === -1) {
+						if(node.owner != -1) {
+							nodeWinner = node.owner;
+						}
 					}
-					unitsUncontested = false;
-				}
-				for(var owner in node.units) {
-					if(owner != node.owner && node.units[owner] > 0) {
+					else {
 						if(gd.debug) {
-							console.log("contested due to units");
+							console.log("contested due to node owner");
+							console.log(nodeWinner, node.owner, typeof nodeWinner, typeof node.owner);
 						}
 						unitsUncontested = false;
 					}
+					for(var unitOwner in node.units) {
+						if(owner != node.owner && node.units[owner] > 0) {
+							if(gd.debug) {
+								console.log("contested due to units");
+							}
+							unitsUncontested = false;
+						}
+					}
 				}
-				if(node.owner != -1) {
+				if(node.owner != -1 && node.owner != -3) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2651546515/issues/source?bid=3248399&fileBranchId=3307983#l765)